docs/testing/unit-tests.md

```diff
@@ -1,3 +1,160 @@
 # Unit Tests
 
-TBD
+As well fostering code quality, well-written unit tests provide a fine way for developers to quickly understand how to run a piece of code, and what it does under various conditions. It's important therefore to maintain high standards not just in our source code, but in our tests as well. 
+
+This guide is intended as a quick reference of common tools and conventions we use for Calypso unit testing. You'll be able to glean a great deal from browsing through Calypso's existing tests, but we'll try to keep this page up to date with the most recent standards.
+
+## Table of contents
+* [Running tests](running-tests)
+* [Writing tests](writing-tests)
+    * [Describing tests](#describing-tests)
+    * [Folder structure](#folder-structure)
+    * [Mocking HTTP requests with nock](#mocking-http-requests-with nock)
+
+## What should I unit test?
+We assume you've Googled the 'why' of unit testing, but sometimes deciding __what__ to unit test is more difficult. 
+
+If your code were to break or receive unexpected input, what side-effects would the rest of the application experience?
+
+## Running tests
```

Comment:
This section is mostly covered with the testing-overview but with improvements - maybe move the improvements there and link?